ecosystem/resources/members/mqt-predictor.toml

```
@@ -0,0 +1,12 @@
+name = "MQT Predictor"
+url = "https://github.com/cda-tum/mqt-predictor"
+description = "MQT Predictor allows one to automatically select a suitable quantum device for a circuit and provides an optimized compiler for the selected device."
+licence = "MIT license"
+contact_info = "quantum.cda@xcit.tum.de"
+affiliations = "Technical University of Munich (TUM), Germany"
+labels = [ "Application package", "Circuit building tool", "Converter", "Machine learning", "Software development kit", "Optimization", "Paper implementation",]
```

```suggestion
labels = [ "Circuit building tool", "Converter", "Machine learning", "Software development kit", "Paper implementation",]
```

I've removed a couple of these: "Application package" and "Optimization" are really for applications outside of quantum computing.